app/views/users/new.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Users#new</h1>
+<p>Find me in app/views/users/new.html.erb</p>
```

Comment:
This view is the generated view. Currently, a user cannot be added via the UI because the form is not here.

Reply:
I will make sure I don't commit these in the future. In the meantime, this is resolved over in my other branch (as it's being used now).